src/Frontend/Views/Gameover.tsx

```diff
@@ -20,31 +20,37 @@ export function Gameover() {
   const winners = uiManager.getWinners();
   const gameDuration = uiManager.getGameDuration();
   const gameover = useGameover();
-  if (!gameover) {
+  if (!gameover || !winners) {
```

Comment:
I think winners will always be true because empty array = true in JS. can you confirm?